src/Maker/MakeEntity.php

```diff
@@ -738,6 +761,63 @@ private function askRelationType(ConsoleStyle $io, string $entityClass, string $
         return $io->askQuestion($question);
     }
 
+    private function askValidationConstraint(ConsoleStyle $io, bool $isFirstConstraint)
+    {
+        if ($isFirstConstraint) {
+            $questionText = 'Which constraint do you want to add?';
+        } else {
+            $questionText = 'Add another constraint?';
+        }
+
+        $question = new Question("$questionText Enter the constraint name (enter <comment>?</comment> to see available constraints or press <return> to stop adding constraints)");
+        $constraintsMap = ConstraintRegistry::getValidationConstraintsMap();
+        $question->setAutocompleterValues($constraintsMap);
+
+        $constraint = $io->askQuestion($question);
+
+        if ('?' === $constraint) {
+            $io->listing(array_keys($constraintsMap));
+
+            return $this->askValidationConstraint($io, $isFirstConstraint);
+        }
+
+        $constraintClass = $constraintsMap[$constraint] ?? $constraint;
+
+        if ($constraintClass) {
+            $options = [];
+
+            $option = null;
+
+            while (true) {
+                if (!$options) {
+                    $option = $io->ask('Do you want to add options to the constraint? Enter the value in key=value format (e.g min=3 or press <return> to stop adding options)');
+                } else {
+                    $option = $io->ask('Add another option? Enter the value in key=value format (e.g min=3 or press <return> to stop adding options)');
+                }
+
+                if (null === $option) {
+                    break;
+                }
+
+                $parts = explode('=', $option);
+
+                if (2 !== \count($parts)) {
+                    $io->error('Attribute format is not correct. Please add options in the format key=value (e.g min=3)');
```

Comment:
Constraints are not always validated as having their config in the `key=value` format, ie. [@Expression](https://symfony.com/doc/current/reference/constraints/Expression.html). I suppose it could be misleading to developers new to Symfony, so maybe we should change example and handle both cases in validation?